tests/core/tests/test_resources.py

```diff
@@ -1188,3 +1188,62 @@ def check_value(self, result, export_headers, expected_value):
         diff = result.rows[0].diff
         self.assertEqual(diff[export_headers.index("categories")],
                          expected_value)
+
+
+@mock.patch("copy.deepcopy")
+@mock.patch("import_export.resources.Diff", spec=True)
+class SkipDiffTest(TestCase):
+    """
+    Tests that the meta attribute 'skip_diff' means that no diff operations are called.
+    """
+    def setUp(self):
+        class _BookResource(resources.ModelResource):
+            class Meta:
+                model = Book
+                skip_diff = True
+        self.resource = _BookResource()
+        self.dataset = tablib.Dataset(headers=['id', 'name', 'birthday'])
+        self.dataset.append(['', 'A.A.Milne', '1882test-01-18'])
+
+    def test_skip_diff(self, mock_diff, mock_deep_copy):
+        result = self.resource.import_data(self.dataset, raise_errors=False)
+        mock_diff.return_value.compare_with.assert_not_called()
+        mock_diff.return_value.as_html.assert_not_called()
+        mock_deep_copy.assert_not_called()
+
+    def test_skip_diff_for_delete_new_resource(self, mock_diff, mock_deep_copy):
+        class BookResource(resources.ModelResource):
+
+            def for_delete(self, row, instance):
+                return True
+
+            class Meta:
```

Comment:
@matthewhegarty picky comment: please follow Django style and put Meta at top (https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/)

Reply:
Done